docker/relay/run-httpd.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Make sure we're not confused by old, incompletely-shutdown httpd
+# context after restarting the container.  httpd won't start correctly
+# if it thinks it is already running.
+rm -rf /run/httpd/* /tmp/httpd*
```

Comment:
these could maybe be tmpfs, which would make it be reset automatically at each restart?

Reply:
I had an issue of starting httpd during docker compose execution so i tried this solution to delete temporary files but now it is useless as httpd is installed with relayd